src/routes/docs/products/ai/integrations/togetherai/+page.markdoc

```diff
@@ -0,0 +1,324 @@
+---
+layout: article
+title: Integrating Together AI
+description: Learn how to integrate Together AI into your Appwrite project.
+difficulty: intermediate
+readtime: 15
+---
+
+Integrating Together AI into your Appwrite project is simple. This tutorial will guide you through the process of setting up the Together AI API and integrating it into your Appwrite project.
+
+# Prerequisites {% #prerequisites %}
+
+- An Appwrite Project
+- A [Together AI API Key](https://docs.together.ai/reference/authentication-1)
+
+{% section #step-1 step=1 title="Create new function" %}
+Head to the [Appwrite Console](https://cloud.appwrite.io/console) then click on **Functions** in the left sidebar and then click on the **Create Function** button.
+
+{% only_dark %}
+![Create function screen](/images/docs/functions/dark/template.png)
+{% /only_dark %}
+
+{% only_light %}
+![Create function screen](/images/docs/functions/template.png)
+{% /only_light %}
+
+1. In the Appwrite Console's sidebar, click **Functions**.
+1. Click **Create function**.
+1. Under **Connect Git repository**, select your provider.
+1. After connecting to GitHub, under **Quick start**, select the **Node.js** starter template.
+1. In the **Variables** step, add the `TOGETHER_API_KEY`, generate it [here](https://api.together.xyz/settings/api-keys).
+1. Follow the step-by-step wizard and create the function.
+{% /section %}
+
+{% section #step-2 step=2 title="Add Undici" %}
+Once the function is created, clone the function and open it in your development environment.
+
+Now you have the repository open, we're going to install `undici` to interact with Together AI's API.
+
+```
+npm install undici
+```
+{% /section %}
+
+{% section #step-3 step=3 title="Create utility function" %}
+For our example, our function will be able to take both `GET` and `POST` requests.
+
+For the `GET` request, return a static HTML page which we'll write later that will use AlpineJS to make a `POST` request to our function.
+Meanwhile, our `POST` request will use the Together AI SDK to make a request to the Together AI API.
+
+To begin with we will write the code to return the static HTML page, to do this we'll create a new `src/utils.js` file with the following code:
+
```

Comment:
```suggestion
To begin with, we need to write the code to return a static HTML page. Create a new `src/utils.js` file with the following code:

```

Lets make sure we use appropriate punctuations . I see that this mistake exists on the Eleven Labs tutorial as well. Please fix it there too